test/jest/jest.fetch.setup.js

```diff
@@ -2038,4 +2038,65 @@ beforeEach(() => {
       status: 416,
     }
   })
+
+  /**
+   * Content set mocks
```

Comment:
Can't we create those mocks in the corresponding test files? This file is getting really cluttered with mocks from all the test suites.

`jest.fetch.setup.js` is being used in `setupFilesAfterEnv` from `jest.config.js`. This is meant for logic which impact all tests.

Reply:
I'm a node novice, so followed the existing pattern, where everything is in jest setup.
What's the best way to move these mocks to the test files.  My naive attempt resulted in a 'No fallback' error.  Reading jest docs to see what I need to do, but if give me a hint, I'd appreciate it.